contributing/DEVELOPING.md

```diff
@@ -84,6 +84,22 @@ In particular, we try to maintain "it might not be perfect but isn't broken"-sup
 
 We also recommend scanning the [spec](../spec/). The non-element part should help understand some of the design aspects.
 
+## Code Reviewes
+
+Before any code can be merged into the AMP repository, it has to be reviewed and approved by the [owner and a core committer](https://github.com/ampproject/amphtml/blob/master/contributing/owners-and-committers.md).
+
+- If you are contributing a fix or a new feature to AMP, be sure to follow the tips on [preparing your code for review](https://github.com/ampproject/amphtml/blob/master/CONTRIBUTING.md#contributing-code-for-a-feature-coding--implementation-phase) and follow [instructions](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#send-a-pull-request-ie-request-a-code-review) on how to submit a pull request.
+
+- Pull requests in AMP are triaged similar to Issues and within 2 days of submitting a PR, someone will be assigned as the reviewer. If you have been working with a core committer or someone else already, @mentioning them in the PR description will be helpful.
+
+- After a reviewer is assigned, expect comments on your code within a few days. Feel free to @mention the reviewer if your PR is high priority or if you don't hear back after 5 days since opening the PR.
```

Comment:
Would it be realistic if we reduce this to 2 business days?  I don't think it's unreasonable to ask that reviewers at least acknowledge the PR, even if it's to say "Thanks for the PR, I will get to it in 3 days."?

Reply:
+1

Reply:
it is reasonable, part of that would be educating the team to make sure when a PR is assigned to someone via triaging, they would either do the review or add a comment with expected timeline within a day. Historically we haven't been good at this. After this is merged, I will send a PSA email to the team.